README.md

```diff
@@ -0,0 +1,81 @@
+# Checkly Terraform provider
+
+[![CircleCI](https://circleci.com/gh/tnolet/terraform-provider-checkly/tree/master.svg?style=svg)](https://circleci.com/gh/tnolet/terraform-provider-checkly/tree/master)
+
+This Terraform provider enables users to manage [Checkly](https://checklyhq.com) resources like checks.
+
+## Using the provider
+
+1. Download the binary applicable for your platform from the [latest tagged release](https://github.com/tnolet/terraform-provider-checkly/releases)
+
+```bash
+
+curl -OL https://github.com/tnolet/terraform-provider-checkly/releases/latest/download/terraform-provider-checkly_darwin_amd64.gz
+```
+
+2. Copy the binary to your Terraform plugin folder, unzip it and rename it to just `terraform-provider-checkly`
+
+```bash
```

Comment:
This is a bit fiddly; could we just include this script in the repo, and give people instructions to run a 'curl |sh'-type command?